src/app/header/header.component.scss

```diff
@@ -1,7 +1,18 @@
 @import "../styles/main";
+#header {
+  background: $blue;
+}
 
 .header-bar {
     background: $header-background;
     margin-top: 0;
     margin-bottom: 0;
 }
+.icons {
+  color: $white;
+  font-size: $navbar-right-icon-size;
```

Comment:
Again, these variables need to be as generic as possible to allow re usability.

Reply:
Actually I tried with the available two fonts-sizes. But either the icons became too small or too large :sweat_smile:

Reply:
@lunayach Add a third font-size for mid range font-sizes

Reply:
@Raxerz Okay! Done.